docs/experiment-analysis/data-pipeline.md

```diff
@@ -44,4 +44,17 @@ Funnel metrics require their own pipeline because of the complicated logic that
 
 1. **Incremental funnel metric events**: We incrementally join assignment data to the event sources for your funnel metrics.
 2. **Funnel metrics data frame**: We generate a data frame for each subject and day for each funnel metric.
-3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
\ No newline at end of file
+3. **Funnel metrics summaries**: We summarize the values for your funnel metrics.
+
+## Privacy
+
+As we’ve detailed, Eppo doesn’t export individual data from your warehouse. We actively minimize what information we store.
+
+* **Fact creation**: We query small sample of tables to help you set-up and make sure the column names and types match. We do not send those results beyond your analysts’ browser nor do we retain them; we just store the configuration. We can anonymize that step by only showing the data type, not the exact value. We do not need access, and encourage you *not* to give our service account access to any personally identifiable information (PII), just unique identifiers that you have generated.
+* **Incremental pipelines**: Our pipelines query daily aggregate of assignments and facts, we only store those on your warehouse. We only keep a copy of the text of the queries that we generate and run (where table and column names appear, but no individual information). We do not keep individual outputs for any daily query. We add them up into incremental tables and store those in your warehouse too.
+* **Auditability**: Our queries and the resulting tables are fully auditable by your data engineers. Our process rely on a service account whose access you control and can revoke at any time. Your engineers configure the service account connection via our web app with no intermediaries. We store the credentials directly into Google Secret Manager. You can update credentials on your own if needed.
+* **Caching results**: We copy and cache overall results (things like incremental metric totals) to make our interface snappy. This is any total that you can see on our service: total number of assignments, total number of conversions, total amounts. For ratios, we store the numerator and denominator (for incrementality).
+
+We keep the interactions between your collaborators with our service in our database: feature flag values, when they were changed, metric definition, who edited what metric, when an experiment was created, etc.
```

Comment:
Do we track who edited metrics? I guess in session replay, but I don't think there is a change log of metric definitions in postgres

We should also mention that we log who ran every query in the ~Fact creation~ (we should rename this SQL definition creation)